lib/block-patterns/three-images-gallery.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Three columns with offset images.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Three columns with offset images', 'Block pattern title', 'default' ),
+	'categories'  => array( 'gallery' ),
+	'content'     => '<!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"25%"} -->
+	<div class="wp-block-column" style="flex-basis:25%"><!-- wp:image {"id":240,"sizeSlug":"large","linkDestination":"none","className":"is-style-default"} -->
```

Comment:
IDs should not be present since they are contextual to the site

Reply:
the image block crashes if I remove this id or make it null

Reply:
https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/block-patterns/large-header.php we have one block pattern on core that is using an ID too

Reply:
we should fix this :)

Reply:
but if I remove the id or make it null it crashes, so what's the proper way to do it? change the block?

Reply:
sorry, not here in the patterns, I mean generally in the image block

Reply:
My mistake, the block does allow this, I was just not deleting the extra class with the ID and that was failing. I removed the ids on all templates